R/dettl.R

```diff
@@ -134,6 +137,10 @@ DataImport <- R6::R6Class(
       invisible(TRUE)
     },
 
+    get_mode = function() {
```

Comment:
This appears untested and/or unused - could you test or remove?

Reply:
Crap added that when I was debugging a thing - forgot to remove it. Done that now - thanks.